controllers/apply-next/form-router.js

```diff
@@ -147,6 +142,34 @@ function initFormRouter(formModel) {
         }
     });
 
+    // Allow an application owned by this user to be deleted
+    router
+        .route('/delete/:applicationId')
+        .get((req, res) => {
+            if (req.params.applicationId && req.user.userData.id) {
```

Comment:
Should we look up the application here first? Only render this screen if the user owns the application?